src/app/components/mno-apps/mno-app.coffee

```diff
@@ -38,13 +38,24 @@ angular.module 'mnoEnterpriseAngular'
 
     vm.pricedPlan = ProvisioningHelper.pricedPlan
 
-    getPricingPlans = () ->
+    # Ordering is possible if
```

Comment:
Consider refactoring? 

```
    orderPossible = (plans, product) ->
      if product
        vm.orderPossible = plans.default[0] || plans[currency] || ProvisioningHelper.skipPriceSelection(product)
      else
        vm.orderPossible = plans.default[0] || plans[currency]

    getPricingPlans = (product) ->
      if vm.isPriceShown
        plans = vm.app.pricing_plans
        vm.pricing_plans = plans[vm.currency]
        orderPossible(plans, product)
```

Reply:
For `orderPossible` I don't think the `plans[currency]` part is necessary. If any plans exist at all then there will always be a `plans.default[0]`.

For `getPricingPlans` it's probably ok as is. It's more explicit with the if statement as it makes it clear that we're checking if the order is possible *because* there was no `plans[vm.currency]`